kuma/wiki/tests/test_views.py

```diff
@@ -3413,9 +3413,10 @@ def test_trailing_slash(self):
         d.title = 'FooFoo'
         d.save()
         mt_url = '/cn/%s/' % (d.slug,)
-        resp = self.client.get(mt_url)
-        eq_(301, resp.status_code)
-        eq_('http://testserver%s' % d.get_absolute_url(), resp['Location'])
+        resp = self.client.get(mt_url, follow=True)
+        eq_(200, resp.status_code)
+        # Check the last redirect chain url is the expected url
+        eq_('http://testserver%s' % d.get_absolute_url(), resp.redirect_chain[-1][0])
```

Comment:
I think this test should be removed, since it no longer applies.  It is testing the ``MINDTOUCH_PROBLEM_LOCALES`` branch which is no longer used, according to code coverage and manual testing.  ``MINDTOUCH_PROBLEM_LOCALES`` and the branch in ``mindtouch_to_kuma_redirect`` that deals with it should be removed as well.

https://github.com/mozilla/kuma/blob/c6c6bb0ae8493d71fb77dd1d1948f2c051394720/kuma/wiki/views/legacy.py#L23-L28

https://github.com/mozilla/kuma/blob/c6c6bb0ae8493d71fb77dd1d1948f2c051394720/kuma/wiki/views/legacy.py#L94-L104